src/Microsoft.OData.Core/Evaluation/LiteralFormatter.cs

```diff
@@ -226,6 +226,11 @@ private static string FormatRawLiteral(object value)
                 return XmlConvert.ToString((DateTimeOffset)value);
             }
 
+            if (value is DateTime)

```

Comment:
During the UT to write a DateTime value to the Uri, this code path needs to be executed in Core as well in order for this to work, when the parameter is DateTime. So I removed the conditional build. Please advice 1. do we need the requirement of writing DateTime as param in Uri. 2. If it is okay to not restrict it to Client lib in this case.              
requestUri = serializer.WriteUriOperationParametersToUri(requestUri, parameters);